lte/gateway/c/session_manager/SessionCredit.h

```diff
@@ -150,7 +150,7 @@ class SessionCredit {
   GrantTrackingType determine_grant_tracking_type(const GrantedUnits& grant);
 
   bool compute_quota_exhausted(const uint64_t allowed,
-    const uint64_t reported, const uint64_t used, float threshold_ratio) const;
+    const uint64_t used, float threshold_ratio, const uint64_t grantedUnits) const;
```

Comment:
Bit of a nit, but I think the styling is a bit inconsistent with the rest of the file.

```
bool compute_quota_exhausted(
  const uint64_t allowed,
  const uint64_t used,
  float threshold_ratio,
  const uint64_t floor) const;
```

Reply:
While we're here, could we rename this function to `is_quota_exhausted`, and add some docs here to define the params and return value? I see that figuring out what's going on requires checking the implementation at the moment.